zone/mob.cpp

```diff
@@ -4812,6 +4822,83 @@ bool Mob::PlotPositionAroundTarget(Mob* target, float &x_dest, float &y_dest, fl
 	return Result;
 }
 
+bool Mob::PlotBotPositionAroundTarget(Mob* target, float& x_dest, float& y_dest, float& z_dest, float min_distance, float max_distance, bool behind_only, bool front_only, bool bypass_los) {
```

Comment:
All of these bot functions should live in a file called `mob_bots.cpp` even if they share the same `mob.h` header